obal/data/roles/diff_package/tasks/brew.yml

```diff
@@ -0,0 +1,23 @@
+---
+- include_tasks: git_package_info.yml
+
+- set_fact:
+    diff_package_changed: False
+
+- name: 'Look up current version of package in brew'
+  shell: "brew list-tagged --quiet --latest {{ item }} {{ inventory_hostname }} | cut -d' ' -f1"
+  register: brew_package_versions
+  tags:
+    - always
+  with_items: "{{ brew_tags }}"
+
+- set_fact:
+    diff_package_changed: "{{ git_package_version not in item.stdout or diff_package_changed }}"
```

Comment:
Should this be a check for "startswith" (`{{ not item.stdout.startswith(git_package-version) or diff_package_changed }}`)?